pkg/crc/machine/machine.go

```diff
@@ -582,6 +582,8 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 				openshiftVersion = crcBundleMetadata.GetOpenshiftVersion()
 			}
 			openshiftStatus = fmt.Sprintf("Running (v%s)", openshiftVersion)
```

Comment:
Before the check for `Progressing`, I would add a
```
} else if operatorsStatus.Degraded {
			openshiftStatus = fmt.Sprintf("Unknown")
} else if operatorsStatus.Processing {
			openshiftStatus = fmt.Sprintf("Starting")
}
```

Reply:
@cfergeau sure, I thought we are just going to ignore this state for sometime but I then why not  put `Degraded` instead `unknown` ?